compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -1683,14 +1681,14 @@ static bool reduceLongOp(TR::Node * node, TR::Block * block, TR::Simplifier * s,
             return false;
          if (newConversionOp == TR::BadILOp)
             {
-            TR::Node::recreate(node, isUnsigned ? TR::iuneg : TR::ineg);
+            TR::Node::recreate(node, TR::ineg);
             TR::Node::recreate(firstChild, TR::l2i);
             }
          else
             {
             TR::Node * temp = TR::Node::create(TR::l2i, 1, firstChild->getFirstChild());
             firstChild->getFirstChild()->decReferenceCount();
-            TR::Node::recreate(firstChild, isUnsigned ? TR::iuneg : TR::ineg);
```

Comment:
what are the semantics of an iuneg? it doesn't even make sense to negate a simply positive value... Just want to make sure that ineg is the right replacement. iuneg doesn't do something like a bitwise not rather than two's compliment negation does it?

Reply:
I am not sure about the purpose of `TR::iuneg`, but `ineg` and `iuneg` both share the same `evaluator`. @Leonardo2718

Reply:
So the evaluators are treating it the same as `ineg` I just wanted to be sure - the fold to `ineg` should be fine.